README.md

```diff
@@ -38,6 +38,11 @@ The following settings must be configured before running the app:
     is not needed. It's recommended that you eventually configure the site to be able
     to send emails. Those configuration steps can be found [below](#enabling-email).
 
+- `OPEN_DISCUSSIONS_HOSTNAME`
+    
+    Sets the hostname for DOCKER_HOST in setup. Should likely be whatever you set the host to in your /etc/hosts
```

Comment:
Minor nitpick - this is a separate a .env variable from `DOCKER_HOST`, only seems to be used by webpack for building the front end.  "Sets the hostname required by webpack for building the front end"?

Reply:
You're not wrong - I like it! Will update before doing the thing :)